src/components/Announcement/Popups/SimplePopup.tsx

```diff
@@ -0,0 +1,71 @@
+import { Trans } from '@lingui/macro'
+import { ReactNode } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { Box, Text } from 'rebass'
+import styled from 'styled-components'
+
+import { NotificationType } from 'components/Announcement/type'
+import { AutoColumn } from 'components/Column'
+import { CheckCircle } from 'components/Icons'
+import IconFailure from 'components/Icons/Failed'
+import WarningIcon from 'components/Icons/WarningIcon'
+import { AutoRow } from 'components/Row'
+import useTheme from 'hooks/useTheme'
+
+const RowNoFlex = styled(AutoRow)`
+  flex-wrap: nowrap;
+`
+
+export default function SimplePopup({
+  title,
+  summary,
+  type = NotificationType.ERROR,
+  icon,
+  link,
+}: {
+  title: string
+  type?: NotificationType
+  summary?: ReactNode
+  icon?: ReactNode
+  link?: string
+}) {
+  const theme = useTheme()
+  const mapColor = {
+    [NotificationType.SUCCESS]: theme.primary,
+    [NotificationType.WARNING]: theme.warning,
+    [NotificationType.ERROR]: theme.red,
+  }
+  const color = mapColor[type]
+  const mapIcon = {
+    [NotificationType.SUCCESS]: <CheckCircle color={color} size={'20px'} />,
+    [NotificationType.WARNING]: <WarningIcon solid color={color} />,
+    [NotificationType.ERROR]: <IconFailure color={color} />,
+  }
+
+  const navigate = useNavigate()
+  const onClickLink = () => {
+    link && navigate(link)
+  }
+  return (
+    <Box>
+      <RowNoFlex>
```

Comment:
I don't get this component, AutoRow added flex-wrap, and you remove it from AutoRow? Why not use Row in first place?

Reply:
thinking now... old code. will update